main.cpp

```diff
@@ -53,83 +53,83 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 //  1)
 //  Noun: phone
-//  action 1:
-phone.ring();
-//  action 2:
-phone.blocked();
-//  action 3:
+//  action 1: Phone receives a call
+phone.ring(); 
+//  action 2: The phone locks the screen
+phone.locksScreen();
+//  action 3: New message or call
 phone.vibrate();
 //  2)
 //  Noun: boy
-//  action 1:
+//  action 1: Boy starts to run
 boy.run();
-//  action 2:
+//  action 2: The boy falls asleep
 boy.sleep();

```

Comment:
I'm marking this because:
- either your plain-english description is not an action, but is an ability or describes a state
- or the plain-english description is fine but the function name doesn't match.

- [ ] revise